libraries/Microsoft.Bot.Schema/ConversationReferenceEx.cs

```diff
@@ -13,47 +13,18 @@ public partial class ConversationReference
         /// <summary>
         /// Creates <see cref="Activity"/> from conversation reference as it is posted to bot.
         /// </summary>
-        public Activity GetPostToBotMessage()
+        public Activity GetContinuationActivity()
```

Comment:
You could simplify this to an expression bodied method _and_ remove some unecessary casting overhead if you wanted to:

```CSharp
public Activity GetContinuationActivity() => 
    new Activity
    {
        Type = ActivityTypes.Event,
        ... rest of property initializers here ...
    }
```